JPS_BASE_LIB/python_wrapper/setup.py

```diff
@@ -1,31 +1,29 @@
 from setuptools import setup, find_namespace_packages
 
 setup(
-    name='py4jps',
-    version='1.41.0',
-    author='Daniel Nurkowski',
-    author_email='danieln@cmclinnovations.com',
+    name='twa',
+    version='0.0.1',
+    author='Jiaru Bai; Daniel Nurkowski',
+    author_email='jb2197@cam.ac.uk; danieln@cmclinnovations.com',
     license='MIT',
-    python_requires='>=3.5',
-    description="Py4jps is a thin Python wrapper for the TheWorldAvatar project.",
-    url="https://github.com/cambridge-cares/TheWorldAvatar/tree/develop/JPS_BASE_LIB/python_wrapper",
+    python_requires='>=3.8',
+    description="twa is a thin Python wrapper for the TheWorldAvatar project.",
```

Comment:
I'd remove the "thin". And also the redundant "the".

Reply:
Addressed in https://github.com/cambridge-cares/TheWorldAvatar/pull/1149/commits/01a8ee11d9e4caa7e77be0d9fdd9ffd51088d91a